### ISSUE TITLE
How to change items.otb alwaysOnTopOrder / StackOrder

### ISSUE THREAD

**Issue by pedbernardo** [TwistedScorpio/OTHire#234]
Hi Alejandro,

Thank you in advance for you work on distros for old protocols like 7.7.

---

**TL;DR** It's possible to change `alwaysOnTopOrder` / **StackOrder** of items.otb in Tibia 7.7? OTItemEditor 0.2.2 doesn't have that option on interface, do you know any alternative?

---

Those last days I'm trying to discover the best distro for that protocol (7.7), and even though I saw your threads in OTLand recommending to use Nostalrius I still giving a try for tests on OTHire. Seems to be better (like Peonso said) reproducing old tibia formulas, monsters behavior and so on.

That said, I'm trying to edit the sprites, and for that I need to alter items.otb, but I'm limited to use OTItemEditor 0.2.2, and that version don't support alter the prop alwaysOnTopOrder on the UI. And I'm unable to find the source files of that version, only 0.2.3 (for Tibia 7.8) on SVN ([link](https://sourceforge.net/projects/opentibia/files/opentibia%20tools/)). And even so, I have no clue how to add that option and re-compile supporting 7.7 protocol.

**Comment by pedbernardo** [comment]
Found a alternative!

Has a recent build of newer Item Editor adding compatibility to older Tibia Protocols: [ItemEditor fork](https://github.com/danilopucci/ItemEditor).

Pointed out by **danilopucci** in [OTLand Forum](https://otland.net/threads/nostalrius-or-othire.268902/post-2604011).
